src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -280,6 +235,76 @@ public static ConfigDef newConfigDef() {
     return configDef;
   }
 
+  /**
+   * Define HDFS configuration group
+   */
+  private static void newHadoopDef(ConfigDef configDef) {
```

Comment:
moved these statements into a separate method because we have a checkstyle requirement of methods being at most 150 lines.

Reply:
Seems like if another config is added the new method will suffer from the same issue. For things like that, adding an as specific as possible suppression in checkstyle might be reasonable as well.